tests/foreman/cli/test_fact.py

```diff
@@ -140,7 +140,7 @@ def test_positive_facts_end_to_end(
         ), f'Assertion failed: {fact} (expected: {expected_value}, actual: {actual_value})'
 
 
-@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+@pytest.mark.rhel_ver_match('N-1')
```

Comment:
IIRC, N-1 means parametrizing tests for EL9 and EL10, I don't understand why we're including so many tests for EL10, which imo could make our tests redundant, so could you please evaluate it again which tests needs to be parametrized?

Reply:
Most of the updated tests are relevant for RHEL10 so I added that. I have removed rhel 10 paraameterization from couple of tests. Please re-review it.